### DISCUSSION TITLE
cmd window command prompt garbled

### DISCUSSION THREAD

**Discussion by comeday** [Eugeny/tabby#8482] [General]
On one of my computers, tabby's cmd window command prompt was garbled, but on my other computer, it is nomal , which was running Windows 10, I had no idea what the problem was.
![screenshot-20230524-232755](https://github.com/Eugeny/tabby/assets/3361917/c5583337-b7aa-48f6-bc44-1d8064c491de)

**Comment by tarocjsu** [comment]
CMD (clink) or CMD (stock) ?
